atlasclient/utils.py

```diff
@@ -14,29 +14,25 @@
 
 """Utilities for the Python client for the MongoDB Atlas API."""
 
-import logging
-
-from http.client import HTTPConnection
+import json
 
 
 class JSONObject(dict):
-    """Dictionary with dot-notation read access."""
+    """Dictionary object with dot-notation read access."""
     def __getattr__(self, name):
         if name in self:
             return self[name]
-        raise AttributeError('{} has no property named {}}.'.format(
+        raise AttributeError('{} has no property named {}.'.format(
```

Comment:
nit: afaict there are no tests around this behavior

Reply:
added a test for this.